src/controllers/slashCommand.ts

```diff
@@ -1,50 +1,16 @@
 import { Request, Response, NextFunction } from 'express'
-import { parse } from './../services/commandService'
+import { parseGive, validateGive } from './../services/commandService'
 import { Db } from 'mongodb'
 
-function command(req: Request, res: Response, next: NextFunction) {
-    const sampleResponse = {
-        'text': 'Would you like to play a game?',
-        'attachments': [
-            {
-                'text': 'Choose a game to play',
-                'fallback': 'You are unable to choose a game',
-                'callback_id': 'wopr_game',
-                'color': '#3AA3E3',
-                'attachment_type': 'default',
-                'actions': [
-                    {
-                        'name': 'game',
-                        'text': 'Chess',
-                        'type': 'button',
-                        'value': 'chess'
-                    },
-                    {
-                        'name': 'game',
-                        'text': `Falken's Maze`,
-                        'type': 'button',
-                        'value': 'maze'
-                    },
-                    {
-                        'name': 'game',
-                        'text': 'Thermonuclear War',
-                        'style': 'danger',
-                        'type': 'button',
-                        'value': 'war',
-                        'confirm': {
-                            'title': 'Are you sure?',
-                            'text': `Wouldn't you prefer a good game of chess?`,
-                            'ok_text': 'Yes',
-                            'dismiss_text': 'No'
-                        }
-                    }
-                ]
-            }
-        ]
-    }
-    console.log(req.body, req.headers)
-    const commandObj = parse(req.body.text)
-    res.json(sampleResponse)
+interface CustomRequest extends Request {
+    command?: any
```

Comment:
If we know what the command looks like please change type any to interface declared in **commandService.ts**